test/libsolidity/GasCosts.cpp

```diff
@@ -96,17 +97,17 @@ BOOST_AUTO_TEST_CASE(string_storage)
 	compileAndRun(sourceCode);
 
 	if (Options::get().evmVersion() <= EVMVersion::byzantium())
```

Comment:
Shouldn't this need a special case for instanbul now? Or only after the evmone release?

Reply:
Moved to #7740.